docs/docs/artifact-store.md

```diff
@@ -250,22 +247,17 @@ Transfer-Encoding: chunked
 
 ### Automatic Path creation
  
-The path in the asset store is computed that way:
-
-* HEAD Request to the asset
-* if ETag is available, take ETag HTTP Header
-* if ETag is not available (dumb HTTP server), download the resource and compute the Sha-1 hash manually
-* filename of the url remains the same
+The path in the asset store is computed in the following way:
 
-The complete path is {artifact store base}/{ETag or ContentHash}/{filename of asset}
-This effectively will create a path, that is unique to the content of the resource.
-If the same URL holds a different entity, a new path is created.
-The same path is downloaded only once. 
-In other words if the path is available in the artifact store, it is not downloaded again.
+1. HEAD Request to the asset.
```

Comment:
See https://www.w3.org/Protocols/rfc2616/rfc2616-sec9.html Paragraph 9.4 for HEAD requests.